ARM-Samples/Create-Video-Analyzer/README.md

```diff
@@ -0,0 +1,92 @@
+
+# Overview
+
+In this Quick-Start you will create an Azure Analyzer for Media (a.k.a "AVAM") resource by using Arm Template (PREVIEW)
+The resource will be deployed to your subscription and will create the 'AVAM' resource based on parametrers presented on the avam.template file.
+
+> **Note:**
+> this sample is *not* for migrating an existing AVAM account to an ARM-Based AVAM account.
+> For a full documentation on AVAM API, visit the [AVAM Developer Portal](https://aka.ms/avam-dev-portal) page.
+> The current API Version is "2021-08-16-preview". Check this Repo from time to time to get updates on new API Versions.
+
+## Prerequisites
+
+* An Azure Media Service (AMS) account. You can get one for free through the [Create AMS Account](https://docs.microsoft.com/en-us/azure/media-services/latest/account-create-how-to).
+
+## Deploy the sample
+
+----
+
+### Option 1 : Click the "Deploy To Azure Button", and fill in the missing parameters
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fmedia-services-video-indexer%2master%2FARM-Samples%2FCreate-Video-Analyzer%2Favam.template.json)  
+
+----
+
+### Option 2 : Deploy using Power Shell Script
+
+1. Open The [Template File](avam.template.json) file and inspect its content.
+2. Fill in the required parameters (see below)
+3. Run the Following Power Shell commands:
+
+* create new Resource group on the same location as your Azure Media Service account, using the [New-AzResourceGroup](https://docs.microsoft.com/en-us/powershell/module/az.resources/new-azresourcegroup) cmdlet.

```

Comment:
we recommend having the ams and the avam accounts in the same resource group I think, but as this is generic instructions, maybe leave it as is